Backends/PDF/graphics.lisp

```diff
@@ -269,27 +336,23 @@
     (pdf:set-rgb-stroke r g b)))
 
 ;;; Clipping region
-(defgeneric pdf-set-clipping-region (region))
+(defgeneric pdf-set-clipping-region (medium region))
 
-(defmethod pdf-set-clipping-region (region)
+(defmethod pdf-set-clipping-region (medium region)
   ;; this is not yet implemented!
-  #+(or)
   (progn
-    (pdf-add-path region)
-    (pdf:clip-path)))
+    #+nil (print region)
+    (pdf-add-path medium region)
+    ))
 
```

Comment:
please add `#+(or)` instead of `#+nil` and put closing parens in the same line as the last form

`(pdf-add-path medium region)))`